packages/chrome/src/index.tsx

```diff
@@ -15,13 +21,15 @@ import {
   LineHeights,
   FontSizes,
   Space,
+  // @ts-ignore
 } from '@theme-ui/editor'
+import { Theme } from '@theme-ui/css'
 
-const runScript = script =>
+const runScript = (script: any) =>
```

Comment:
```suggestion
const runScript = (script: string) =>
```

https://developer.chrome.com/extensions/devtools_inspectedWindow#method-eval